File: JavaScript/7js 2/quotable.js
/*
|--------------------------------------------------------------------------
| Thinkful - String Drills: Quotable
|--------------------------------------------------------------------------
|
*/

function quotable(name, quote) {
   return `${name} said: "${quote}"`;
}


const quotable = (_, __) => `${_} said: "${__}"`


const quotable = (name, quote) => name + " said: \"" + quote + "\""


const quotable = (name, quote) => {
   return '[name] said: "[quote]"'.replace('[name]', name).replace('[quote]', quote);
}


function quotable(name, quote) {
   return name + ` said: ` + `"` + quote + `"`;
}


console.log(quotable('Grae', 'Practice makes perfect'), 'Grae said: "Practice makes perfect"')
console.log(quotable('Dan', 'Get back to work, Grae'), 'Dan said: "Get back to work, Grae"')
console.log(quotable('Alex', 'Ruby is great fun'), 'Alex said: "Ruby is great fun"')

File: Python/7py 3/quotable.py
"""
|--------------------------------------------------------------------------
| Thinkful - String Drills: Quotable
|--------------------------------------------------------------------------
|
"""

def quotable(name, quote):
   return f'{name} said: "{quote}"'


def quotable(name, quote):
   return '{} said: "{}"'.format(name, quote)


def quotable(name, quote):
   return name + ' said: ' + '"' + quote + '"'


def quotable(_n, _q):
   return ' said: '.join([_n, f'"{_q}"'])


print(quotable('Grae', 'Practice makes perfect'), 'Grae said: "Practice makes perfect"')
print(quotable('Dan', 'Get back to work, Grae'), 'Dan said: "Get back to work, Grae"')
print(quotable('Alex', 'Python is great fun'), 'Alex said: "Python is great fun"')